app/scaffold/scaffold.js

```diff
@@ -181,6 +182,9 @@ class Scaffold {
             if (!promise) {
                 return;
             }
+
+            DOM.alertBox("Decrypting...");
+
             promise
                 .then((decrypted) => {
                     console.log(`Decrypting '${orderId}'`, decrypted);
```

Comment:
Minor, but should this say "Decrypted ${orderId}" to make it past tense? At this point the document has been decrypted right?